web/src/settings_components.ts

```diff
@@ -691,7 +691,7 @@ type setting_property_type =
 export function check_property_changed(
     elem: HTMLElement,
     for_realm_default_settings: boolean,
-    sub: StreamSubscription,
+    sub: StreamSubscription | undefined,
     group: UserGroup,
```

Comment:
I think the same is true for the `group` field.

Reply:
Yes.